models/cart/src/cart/cart-draft/presets/sample-data-fashion/sample-germany-01.ts

```diff
@@ -43,6 +47,7 @@ const sampleGermany01 = (customerId?: string): TCartDraftBuilder =>
     ])
     .shippingMethod(
       KeyReference.presets.shippingMethod().key(shippingMethod.key!)
-    );
+    )
+    .store(KeyReference.random().typeId('store').key(store.key!));
```

Comment:
Is it worth adding stores as a key reference preset?

Reply:
Admittedly I was in a rush, but I can add this. Shouldn't be a problem 👍

Reply:
See e221359e8ec717b0c58189d5eab8038fdb1d1add and e323dcb01bbbb16646d15c1f57eba5498f8ffe0d